src/StatusNotifier/Host/Service.hs

```diff
@@ -354,7 +354,6 @@ build Params { dbusClient = mclient
 
       clientRegistrationPairs =
         [ (I.registerForNewIcon, handleNewIcon)
-        , (I.registerForNewIconThemePath, handleNewIcon)
```

Comment:
Why is this removed?

Reply:
Indeed gnome handles it: https://github.com/ubuntu/gnome-shell-extension-appindicator/blob/master/interfaces-xml/StatusNotifierItem.xml
Restored.